### DISCUSSION TITLE
Accessing Local.Send from the Command Line

### DISCUSSION THREAD

**Discussion by jvaccaro** [localsend/localsend#1498] [Discussion (English, International)]
Hello;
Loving Local.Send! I'd like to send-only messages, and files from a command line in Ubuntu Linux.  I understand handling the encryption might be a challenge.

Mostly wondering if its feasible and if there are any other users interested in this mode of operation.

JohnV
------------------------------------------------
_Only puny secrets need protection. Big discoveries are protected by public incredulity_. - McLuhan

**Comment by Tienisto** [comment]
The CLI is still in progress. See #11
